draft-irtf-cfrg-hpke.md

```diff
@@ -1131,16 +1175,19 @@ the PSK or AuthPSK mode, is proven in {{HPKEAnalysis}}; in a quantum
 setting, the remaining security level is smaller and defined by the
 post-quantum security level of the AEAD scheme.
 
-## Security Requirements on a KEM used within HPKE
+## Security Requirements on a KEM used within HPKE {#kem-security}
+
+The Encap/Decap interface of a KEM used within HPKE MUST be
+IND-CCA2-secure.
+
+The AuthEncap/AuthDecap interface of a KEM used within HPKE MUST be
+Insider-CCA-secure and Insider-Auth-secure. If the application does not
```

Comment:
I'm not sure Insider-CCA-secure and insider-Auth-secure are common terms. We should probably define them.

Reply:
Yes, we should define them somehow. Similar notions have been around in Signcryption literature for a while, but it is true that they are not so commen, and also the notions I am working on with the others are a bit different. Should we define them in an appendix?

Reply:
Eh... maybe? Or inline here?